apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx

```diff
@@ -0,0 +1,187 @@
+      submitButtonText={'Stofna notanda'}
+      onSubmit={submit}
+    >
+      <Box marginBottom={3}>
+        <Input
+          label="Kennitala"
+          name="staffNationalId"
+          type={'number'}
+          placeholder="Sláðu inn kennitölu"
+          backgroundColor="blue"
+          value={state.staffNationalId}
+          onChange={(event) => {
+            if (event.currentTarget.value.length <= 10) {
+              setState({
+                ...state,
+                staffNationalId: event.currentTarget.value,
+                hasError: false,
+              })
+            }
+          }}
+          hasError={
+            state.hasError &&
+            (!state.staffNationalId || state.staffNationalId.length !== 10)
+          }
+        />
+      </Box>
+      <Box marginBottom={3}>
+        <Input
+          label="Nafn"
+          name="staffName"
+          value={state.staffName}
+          placeholder="Sláðu inn fullt nafn"
+          backgroundColor="blue"
+          onChange={(event) => {
+            setState({
+              ...state,
+              hasError: false,
+              staffName: event.currentTarget.value,
+            })
+          }}
+          hasError={state.hasError && !state.staffName}
+        />
+      </Box>
+      <Box marginBottom={2}>
+        <Input
+          label="Netfang"
+          name="staffEmail"
+          value={state.staffEmail}
+          placeholder="Sláðu inn netfang"
+          backgroundColor="blue"
+          type={'email'}
+          onChange={(event) => {
+            setState({
+              ...state,
+              hasError: false,
+              staffEmail: event.currentTarget.value,
+            })
+          }}
+          hasError={
+            state.hasError &&
+            (!state.staffEmail || !isEmailValid(state.staffEmail))
+          }
+        />
+      </Box>
+      <Text marginBottom={5} variant="small">
+        Notandi fær sendan tölvupóst með hlekk til að skrá sig inn með rafrænum
+        skilríkjum.
+      </Text>
+      <Text marginBottom={3} variant="h4">
+        Réttindi notanda
+      </Text>
+      <Box marginBottom={3}>
+        <Checkbox
+          name={'employee'}
+          label="Vinnsluaðili"
+          checked={state.roles.includes(StaffRole.EMPLOYEE)}
+          strong={false}
+          hasError={state.hasError && state.roles.length === 0}
+          onChange={(event) => {
+            changeStaffAccess(StaffRole.EMPLOYEE, event.target.checked)
+          }}
+          filled={false}
+        />
+      </Box>
+      <Box marginBottom={2}>
+        <Checkbox
+          name={'admin'}
+          label="Stjórnandi (admin)"
+          checked={state.roles.includes(StaffRole.ADMIN)}
+          hasError={state.hasError && state.roles.length === 0}
+          onChange={(event) => {
+            changeStaffAccess(StaffRole.ADMIN, event.target.checked)
+          }}
+          strong={false}
```

Comment:
filled is false but not in the other checkbox, is the other one automatically filled? 💭

Reply:
Nope it is default on false, will remove from the top one :)